### PR TITLE
First pass at v3.8 support

### PR DESCRIPTION
**Pull request by sdodson**

### CODE REVIEW

**Comment by sdodson** [comment]
@mtnbikenc PTAL, we should really simplify the addition of new releases but my interest is just getting the CI queue unblocked right now. We can work on making this easier in the future.

**Comment by sdodson** [comment]
tox should be clean now

File: playbooks/byo/openshift-cluster/upgrades/v3_8/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# v3.6 Major and Minor Upgrade Playbook
```
Comment (mtnbikenc) [inline playbooks/byo/openshift-cluster/upgrades/v3_8/README.md]: v3.8

I have a PR open that addresses v3.7 here.

**Comment by sdodson** [comment]
Install is successful with e2e test failures. I'm going ahead and merging this.

**Pull request merged**